src/index.js

```diff
@@ -106,11 +106,11 @@ function ParseServer({
     passwordResetSuccess: undefined
   },
 }) {
-  
+  setFeature('serverVersion', parseServerPackage.version);
```

Comment:
how about we use a `metadata` field to hold the version info. it feels odd to treat it as a feature like the others

id like the payload to be something like:
```
{ 
  metadata: {}
  features: {}
}
```
its prob out of scope so i can make the changes in a later pr